pm/sender.go

```diff
@@ -155,7 +155,9 @@ func (s *sender) validateTicketParams(ticketParams *TicketParams, numTickets int
 	}
 
 	latestBlock := s.timeManager.LastSeenBlock()
-	if ticketParams.ExpirationBlock.Cmp(latestBlock) <= 0 {
+
+	currentBuffer := new(big.Int).Sub(latestBlock, ticketParams.ExpirationBlock).Int64()
```

Comment:
Nit: If we're defining a buffer as the # of blocks before the expiration block could the following be slightly more readable?

```golang
currentBuffer := new(big.Int).Sub(ticketParams.ExpirationBlock, latestBlock).Int64()
if currentBuffer <= paramsExpiryBuffer {
    ...
}
```

Otherwise, the current code evaluates to a negative buffer in the normal case and only evaluates to a positive buffer when `ErrTicketParamsExpired` should be returned.